src/jit/emit.cpp

```diff
@@ -7481,7 +7481,7 @@ regMaskTP emitter::emitGetGCRegsSavedOrModified(CORINFO_METHOD_HANDLE methHnd)
         CorInfoHelpFunc helpFunc = Compiler::eeGetHelperNum(methHnd);
 
         // Get the set of registers that this call kills and remove it from the saved set.
-        regMaskTP savedSet = RBM_ALLINT & ~emitComp->compNoGCHelperCallKillSet(helpFunc);
+        regMaskTP savedSet = RBM_ALLINT & ~emitGetGCRegsKilledByNoGCCall(helpFunc);
```

Comment:
This is the only place where we call `emitGetGCRegsKilledByNoGCCall`, so it would be better to move `RBM_ALLINT & ~call` inside this function, but in this case we would need to rename it to `emitGetGCRegsSavedOrModifiedByNoGCCall` that looks too long.